### ISSUE TITLE
v0.13.11 is not compilable on web

### ISSUE THREAD

**Issue by xvrh** [rive-app/rive-flutter#410]
### Description
Try to compile an app with v0.13.11 (& rive_common v0.4.10)
`flutter build web`

Error message:
```
Target dart2js failed: ProcessException: Process exited abnormally with exit code 1:
../../.pub-cache/hosted/pub.dev/rive_common-0.4.10/lib/src/dynamic_library_helper.dart:1:8:
Error: Dart library 'dart:ffi' is not available on this platform.
import 'dart:ffi';
       ^
Info: The unavailable library 'dart:ffi' is imported through these packages:

    main.dart => package:playground_web => package:rive => package:rive_common => dart:ffi

Detailed import paths for (some of) the these imports:

    main.dart => package:playground_web/main.dart => package:rive/rive.dart => package:rive_common/src/dynamic_library_helper.dart => dart:ffi

../../.pub-cache/hosted/pub.dev/rive_common-0.4.10/lib/src/dynamic_library_helper.dart:14:16:
Error: Type 'DynamicLibrary' not found.
  static final DynamicLibrary nativeLib = open();
               ^^^^^^^^^^^^^^
../../.pub-cache/hosted/pub.dev/rive_common-0.4.10/lib/src/dynamic_library_helper.dart:16:10:
Error: Type 'DynamicLibrary' not found.
  static DynamicLibrary open() {
         ^^^^^^^^^^^^^^
../../.pub-cache/hosted/pub.dev/rive_common-0.4.10/lib/src/dynamic_library_helper.dart:52:10:
Error: Type 'DynamicLibrary' not found.
  static DynamicLibrary _openAndroidDynamicLibraryWithFallback() {
         ^^^^^^^^^^^^^^
../../.pub-cache/hosted/pub.dev/rive_common-0.4.10/lib/src/dynamic_library_helper.dart:14:16:
Error: 'DynamicLibrary' isn't a type.
  static final DynamicLibrary nativeLib = open();
               ^^^^^^^^^^^^^^
../../.pub-cache/hosted/pub.dev/rive_common-0.4.10/lib/src/dynamic_library_helper.dart:25:20:
Error: Undefined name 'DynamicLibrary'.
            return DynamicLibrary.open(
                   ^^^^^^^^^^^^^^
../../.pub-cache/hosted/pub.dev/rive_common-0.4.10/lib/src/dynamic_library_helper.dart:35:20:
Error: Undefined name 'DynamicLibrary'.
            return DynamicLibrary.open(
                   ^^^^^^^^^^^^^^
../../.pub-cache/hosted/pub.dev/rive_common-0.4.10/lib/src/dynamic_library_helper.dart:47:14:
Error: Undefined name 'DynamicLibrary'.
      return DynamicLibrary.open('rive_common_plugin.dll');
             ^^^^^^^^^^^^^^
../../.pub-cache/hosted/pub.dev/rive_common-0.4.10/lib/src/dynamic_library_helper.dart:49:12:
Error: Undefined name 'DynamicLibrary'.
    return DynamicLibrary.process();
           ^^^^^^^^^^^^^^
../../.pub-cache/hosted/pub.dev/rive_common-0.4.10/lib/src/dynamic_library_helper.dart:54:14:
Error: Undefined name 'DynamicLibrary'.
      return DynamicLibrary.open('librive_text.so');
             ^^^^^^^^^^^^^^
../../.pub-cache/hosted/pub.dev/rive_common-0.4.10/lib/src/dynamic_library_helper.dart:66:14:
Error: Undefined name 'DynamicLibrary'.
      return DynamicLibrary.open('/data/data/$appId/lib/librive_text.so');
             ^^^^^^^^^^^^^^
../../.pub-cache/hosted/pub.dev/rive_common-0.4.10/lib/src/dynamic_library_helper.dart:83:5:
Error: Undefined name 'DynamicLibrary'.
    DynamicLibrary.open('librive_text.so');
    ^^^^^^^^^^^^^^
../../.pub-cache/hosted/pub.dev/rive_common-0.4.10/lib/src/dynamic_library_helper.dart:91:5:
Error: Undefined name 'DynamicLibrary'.
    DynamicLibrary.open('librive_text.so');
    ^^^^^^^^^^^^^^
Error: Compilation failed.
  Command: /Users/xavier/projects/flutter_stable/bin/cache/dart-sdk/bin/dart --disable-dart-dev
  /Users/xavier/projects/flutter_stable/bin/cache/dart-sdk/bin/snapshots/dart2js.dart.snapshot
  --platform-binaries=/Users/xavier/projects/flutter_stable/bin/cache/flutter_web_sdk/kernel --invoker=flutter_tool -Ddart.vm.product=true
  -DFLUTTER_WEB_CANVASKIT_URL=https://www.gstatic.com/flutter-canvaskit/235db911ba279722f5e685f38b0ed30fa7e8570a/ -DFLUTTER_WEB_AUTO_DETECT=true
  --native-null-assertions --no-source-maps -o /Users/xavier/projects/playground_web/.dart_tool/flutter_build/9b1a33ccdfb365d646fd0a03d6abbb77/app.dill
  --packages=.dart_tool/package_config.json --cfe-only /Users/xavier/projects/playground_web/.dart_tool/flutter_build/9b1a33ccdfb365d646fd0a03d6abbb77/main.dart
#0      RunResult.throwException (package:flutter_tools/src/base/process.dart:125:5)
#1      _DefaultProcessUtils.run (package:flutter_tools/src/base/process.dart:321:19)
<asynchronous suspension>
#2      Dart2JSTarget.build (package:flutter_tools/src/build_system/targets/web.dart:193:5)
<asynchronous suspension>
#3      _BuildInstance._invokeInternal (package:flutter_tools/src/build_system/build_system.dart:875:9)
<asynchronous suspension>
#4      Future.wait.<anonymous closure> (dart:async/future.dart:524:21)
<asynchronous suspension>
#5      _BuildInstance.invokeTarget (package:flutter_tools/src/build_system/build_system.dart:813:32)
<asynchronous suspension>
#6      Future.wait.<anonymous closure> (dart:async/future.dart:524:21)
<asynchronous suspension>
#7      _BuildInstance.invokeTarget (package:flutter_tools/src/build_system/build_system.dart:813:32)
<asynchronous suspension>
#8      FlutterBuildSystem.build (package:flutter_tools/src/build_system/build_system.dart:642:16)
<asynchronous suspension>
#9      WebBuilder.buildWeb (package:flutter_tools/src/web/compile.dart:92:34)
<asynchronous suspension>
#10     BuildWebCommand.runCommand (package:flutter_tools/src/commands/build_web.dart:230:5)
<asynchronous suspension>
#11     FlutterCommand.run.<anonymous closure> (package:flutter_tools/src/runner/flutter_command.dart:1394:27)
<asynchronous suspension>
#12     AppContext.run.<anonymous closure> (package:flutter_tools/src/base/context.dart:153:19)
<asynchronous suspension>
#13     CommandRunner.runCommand (package:args/command_runner.dart:212:13)
<asynchronous suspension>
#14     FlutterCommandRunner.runCommand.<anonymous closure> (package:flutter_tools/src/runner/flutter_command_runner.dart:372:9)
<asynchronous suspension>
#15     AppContext.run.<anonymous closure> (package:flutter_tools/src/base/context.dart:153:19)
<asynchronous suspension>
#16     FlutterCommandRunner.runCommand (package:flutter_tools/src/runner/flutter_command_runner.dart:308:5)
<asynchronous suspension>
#17     run.<anonymous closure>.<anonymous closure> (package:flutter_tools/runner.dart:130:9)
<asynchronous suspension>
#18     AppContext.run.<anonymous closure> (package:flutter_tools/src/base/context.dart:153:19)
<asynchronous suspension>
#19     main (package:flutter_tools/executable.dart:93:3)
<asynchronous suspension>

Compiling lib/main.dart for the Web...                             13.8s
Error: Failed to compile application for the Web.

```

### Device & Versions (please complete the following information)
```
Flutter 3.22.3 • channel stable • https://github.com/flutter/flutter.git
Framework • revision b0850beeb2 (8 days ago) • 2024-07-16 21:43:41 -0700
Engine • revision 235db911ba
Tools • Dart 3.4.4 • DevTools 2.34.3
```

**Comment by AndreiMisiukevich** [comment]
I confirm. It was working fine yesterday but suddenly broke today.
Unfortunately, the new version of Rive caused it.

**Comment by HayesGordon** [comment]
Hey all, thank you for reporting. We're deploying a fix, and it should be out within the hour.

**Comment by HayesGordon** [comment]
Fix is out in **v0.13.12**

Please reopen this if you're still encountering issues.
